docs/content/routing/services/index.md

```diff
@@ -300,6 +306,34 @@ On subsequent requests, to keep the session alive with the same server, the clie
           sameSite = "none"
     ```
 
+??? example "Adding Stickiness with domain defined -- Using the [File Provider](../../providers/file.md)"
```

Comment:
I am not sure this example is needed, as the previous one already documents how to use custom cookie options. If you want to document the cookie domain, why not adding `domain: example.com` to the previous example?